src/cashlib/cashlib.cpp

```diff
@@ -76,24 +110,13 @@ SLAPI int Bin2Hex(unsigned char *val, int length, char *result, unsigned int res
     return 1;
 }
 
-
-/** Return random bytes from cryptographically acceptable random sources */
-SLAPI int RandomBytes(unsigned char *buf, int num)
-{
```

Comment:
This remove RandomBytes from cashlib for all targets not just android. I don't think it is correct?

Reply:
Its just moved lower in the file so I can provide a different Android implementation.